modmesh/pilot/_svg_gui.py

```diff
@@ -92,18 +92,35 @@ def _get_initial_path():
         return found
 
     def _load_svg_file(self, filename):
-        parser = svg.PathParser(filename)
-        parser.parse()
+        path_parser = svg.PathParser(filename)
+        path_parser.parse()
+
+        shape_parser = svg.ShapeParser(filename)
```

Comment:
Use `ShapeParser` to parse shapes except `Path`. We may want to refactor the SVG parser in the future. So far, I think the current way is enough to work.